addons/hitreactions/XEH_postInit.sqf

```diff
@@ -0,0 +1,50 @@
+#include "script_component.hpp"
+
+if (GVAR(weaponDropChanceGunHit) + GVAR(weaponDropChanceArmHit) == 0) exitWith {};
+
+["ace_firedNonPlayer", {
+    params ["", "", "", "", "", "", "_projectile"];
+    private _roll = random 1;
+    if (!(_roll < GVAR(weaponDropChanceGunHit) || _roll < GVAR(weaponDropChanceArmHit))) exitWith {};
+    _projectile addEventHandler ["HitPart", {
+        params ["_projectile", "_hitEntity", "_projectileOwner", "_pos", "_velocity", "_normal", "_components", "_radius" ,"_surfaceType"];
+        [_surfaceType, _components, _pos] call FUNC(checkWeaponDrop);
+    }];
+}] call CBA_fnc_addEventHandler;
+
+if (!hasInterface) exitWith {};
+
+["ace_firedPlayer", {
+    params ["", "", "", "", "", "", "_projectile"];
+    private _roll = random 1;
+    if (!(_roll < GVAR(weaponDropChanceGunHit) || _roll < GVAR(weaponDropChanceArmHit))) exitWith {};
+    _projectile addEventHandler ["HitPart", { 
+        params ["_projectile", "_hitEntity", "_projectileOwner", "_pos", "_velocity", "_normal", "_components", "_radius" ,"_surfaceType"];
+        [_surfaceType, _components, _pos] call FUNC(checkWeaponDrop);
+    }];
+}] call CBA_fnc_addEventHandler;
+
+[QGVAR(dropGun), {
```

Comment:
This can happen locally to a machine without an interface. You need to add tihs EH to all machines.